clients/python/agentic-sandbox-client/pyproject.toml

```diff
@@ -22,6 +22,15 @@ dependencies = [
 "Homepage" = "https://github.com/kubernetes-sigs/agent-sandbox"
 "Bug Tracker" = "https://github.com/kubernetes-sigs/agent-sandbox/issues"
 
+# Define the "tracing" optional dependency group
+[project.optional-dependencies]
+tracing = [
+    "opentelemetry-api",
```

Comment:
The new OpenTelemetry dependencies are not version-pinned. This could lead to non-reproducible builds or break the client if a future release of one of these libraries introduces breaking changes. Please pin the dependencies to specific, known-good versions (e.g., `opentelemetry-sdk == 1.25.0`).